ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -221,17 +220,15 @@ snpe_subplugin::runtimeToString (zdl::DlSystem::Runtime_t runtime)
  * @brief Internal method to get names of output layers from tensors information.
  */
 bool
-snpe_subplugin::set_output_layer_names (const GstTensorsInfo *info)
+snpe_subplugin::set_output_tensor_names (const GstTensorsInfo *info)
```

Comment:
do you need to initialize (clean up) ```output_tensor_names_list``` before the loop?

Reply:
Added checking size in this function, PTAL.